flymake-vale.el

```diff
@@ -150,24 +173,24 @@ Passing the results and source BUF to CALLBACK."
   (when (process-live-p flymake-vale--proc)
     (kill-process flymake-vale--proc))
   (let* ((process-connection-type nil)
+         (callback flymake-vale--report-fnc)
+         (buf (current-buffer))
```

Comment:
I believe you've combined one of my later commits along with your work.

Reply:
Oops! I'll untangle & force-push